### ISSUE TITLE
Cluster gets (re)created when running amp cluster rm

### ISSUE THREAD

**Issue by alasdairhurst** [appcelerator-archive/amp#1402]
**Description**

When running ```amp -s localhost cluster rm``` after creating a cluster, the cluster does not get removed. The logs that are output are the same as when running ```amp -s localhost cluster create```
Windows 10 Anniversary x64, docker 17.03.1, amp v0.12.0-dev

**Steps to reproduce the issue:**
1. create a cluster  ```amp -s localhost cluster create```
2. remove the cluster  ```amp -s localhost cluster rm```

**Describe the results you received:**
```
$ ./amp -s localhost cluster rm
[localhost:50101]
bootstrapping cluster
wait for cluster
ok
deploy amp monitoring stack to cluster - stage 1
deploy ampmon.1.stack.yml => amp
Updating service amp_elasticsearch (id: nvxesm4ys8fcs2c3qargenb4s)
Updating service amp_nats (id: d8e0do0gh68abao68xjd371ak)
Updating service amp_etcd (id: q1sl2vgxnqg50melmxt83il20)
ok
wait for all amp monitoring stage 1 stack service replicas to be running
ok
deploy amp monitoring stack to cluster - stage 2
deploy ampmon.2.stack.yml => amp
Updating service amp_ampbeat (id: w4c0mibn1qwnf22nack8lx7j5)
Updating service amp_kibana (id: 5as277v0swy49y3cjrdkilyjo)
ok
wait for all amp monitoring stage 2 service replicas to be running (540 sec)
ok
deploy amp stack to cluster
deploy amp.stack.yml => amp
Updating service amp_amplifier-gateway (id: qiibd3h9l0tlbapxfz4vhtm6y)
Updating service amp_agent (id: mwk1145u0vzravak6gidx1uwi)
Updating service amp_amplifier (id: z9upbi71y4myq1t2sl1i3i9jv)
ok
wait for all amp service replicas to be running (540 sec)
ok
Generating RSA private key, 2048 bit long modulus
............................+++
................................................+++
e is 65537 (0x10001)
Generating RSA private key, 2048 bit long modulus
............................+++
..+++
e is 65537 (0x10001)
Signature ok
subject=/C=US/ST=California/L=Santa Clara/O=Axway/OU=Atomiq/CN=dashboard.local.atomiq.io
Getting CA Private Key
588b1e61b81b8f4cc004a3ef212ae810b60303a5b2fa184e1cfd2730d29db673
checking if the certificate secret is already defined
deploy amp proxy stack to cluster
deploy ampproxy.stack.yml => amp
Updating service amp_proxy (id: yb9q9t0inga4rmfbvem1tv603)
ok
wait for all amp proxy service replicas to be running (540 sec)
ok

Cluster status: healthy

```

**Describe the results you expected:**

cluster was deleted

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
$ docker version
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      windows/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Fri Mar 24 00:00:50 2017
 OS/Arch:      linux/amd64
 Experimental: true

```

**Docker for Mac/Windows version, if applicable:**

```
Version 17.03.1-ce-win12 (12058)
```

**Output of `amp version`:**

```
$ ./amp version
[cloud.appcelerator.io:50101]

Client:
 Version:       v0.12.0-dev
 Build:         056e5e9f
 Server:        cloud.appcelerator.io:50101
 Go version:    go1.8.1
 OS/Arch:       windows/amd64

Server:         not connected
```

**Additional environment details (AWS, etc., if applicable):**

**Comment by ingo** [comment]
It appears the proper way to delete (for now):

docker stack rm pinger
docker stack rm amp

cc @bladedancer

**Comment by ndegory** [comment]
related to #1212 
this will be fully fixed in next release, and right now if fixed in 0.12.0-dev when using the option -t latest (this uses the latest pushed version of the images, in particular the amp-bootstrap image related to this issue).

**Comment by JosephGJ** [comment]
fixed with 0.12.0 release
